starter-code/index.html

```diff
@@ -2,30 +2,85 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  Instagram
+  
+    <div class="background">
+     <img src="./images/bgtile.png" alt="">
```

Comment:
using a div like this will make you design a bit weird (as you may have notice) as it's a block element, all your content will be under this div.

Instead, you can use this image (bgtile.png) as the background image of your body, like this : 

```css
body {
  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
  margin: 0;
  background-image: url("../images/bgtile.png");
  background-repeat: repeat-x;
  background-color: #f6f7f9;


}
```